cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -44,6 +49,5 @@ gen.add("tolerance",          double_t, 0, "Tolerance around joint position limi
 gen.add("base_compensation",  bool_t,   0, "Compensate base motion",  False)
 gen.add("base_active",        bool_t,   0, "Command base actively",  False)
```

Comment:
replace "base_active" by an kinematic_extension_enum with values None, BaseActive (and later LookAt)

take care of not setting base_compensation and kinematic_extension.base_active at the same time!!!